test/fixtures/generator/bwb_spec.json

```
@@ -11,6 +11,23 @@
         "VALIDATE_WORKSPACE": false
     },
     "configuration": "darwin_x86_64-fastbuild-ST-d53d69b6b8c1",
+    "custom_xcode_schemes": [
+        {
+            "build_action": null,
+            "launch_action": {
+                "args": null,
+                "env": null,
+                "target": "@//tools/generator:generator",
+                "working_directory": null
+            },
+            "name": "Generator",
+            "test_action": {
+                "targets": [
+                    "@//tools/generator/test:tests"
```

We should remove the `@` if it's not an external repo, to match the way Bazel canonicalizes these, and allows for easier matching to `Target.label`.

```suggestion
                    "//tools/generator/test:tests"
```

---

OK. That is the value that comes back from `native.repository_name()`

---

Fixed in #664.